App fails to serve (ng serve) post-upgrade to ng-whiteboard 5.0

### Describe the bug
After upgrading from v4.x to v5.0 of ng-whiteboard, and applying the required changes for v5.0, the application fails to start. If I import NgWhiteboardComponent into one of my components, the build/serve process crashes with “resolve” errors.

### Expected behavior
The application should build and serve successfully without crashing when NgWhiteboardComponent is imported.

### Screenshots or Video

<img width="2564" height="1283" alt="Image" src="https://github.com/user-attachments/assets/73785f98-22d6-424a-bd58-5148524cc1b4" />

### Environment
- Angular version: 20.3.11
- ng-whiteboard version: 5.0.0
- Browser (if relevant): Chrome (latest)
- OS: Fedora 43
- Package manager: pnpm 10.20.0

### Additional context
```
[ng] ✘ [ERROR] Could not resolve "@core/event-bus"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:4:20:
[ng]       4 │ import * as i1 from '@core/event-bus';
[ng]         ╵                     ~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/event-bus" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/config/config.service"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:7:49:
[ng]       7 │ ...figService as ConfigService$1 } from '@core/config/config.service';
[ng]         ╵                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/config/config.service" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/directives"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:8:64:
[ng]       8 │ ...andlerDirective, GlobalKeyboardDirective } from '@core/directives';
[ng]         ╵                                                    ~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/directives" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/svg/svg.directive"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:9:29:
[ng]       9 │ import { SvgDirective } from '@core/svg/svg.directive';
[ng]         ╵                              ~~~~~~~~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/svg/svg.directive" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/svg/svg.service"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:10:43:
[ng]       10 │ import { SvgService as SvgService$1 } from '@core/svg/svg.service';
[ng]          ╵                                            ~~~~~~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/svg/svg.service" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/types"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:11:69:
[ng]       11 │ ...ype as ElementType$1, ToolType as ToolType$1 } from '@core/types';
[ng]          ╵                                                        ~~~~~~~~~~~~~
[ng]   You can mark the path "@core/types" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/api"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:12:43:
[ng]       12 │ import { ApiService as ApiService$1 } from '@core/api';
[ng]          ╵                                            ~~~~~~~~~~~
[ng]   You can mark the path "@core/api" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/canvas"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:13:49:
[ng]       13 │ import { CanvasService as CanvasService$1 } from '@core/canvas';
[ng]          ╵                                                  ~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/canvas" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/elements"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:14:55:
[ng]       14 │ ... { SelectionService as SelectionService$1 } from '@core/elements';
[ng]          ╵                                                     ~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/elements" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/pipes"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:15:69:
[ng]       15 │ ...orPipe, ElementOpacityPipe, PointsToPathPipe } from '@core/pipes';
[ng]          ╵                                                        ~~~~~~~~~~~~~
[ng]   You can mark the path "@core/pipes" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/tools"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:16:47:
[ng]       16 │ import { ToolsService as ToolsService$1 } from '@core/tools';
[ng]          ╵                                                ~~~~~~~~~~~~~
[ng]   You can mark the path "@core/tools" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] /mnt/WorkGames/work/abodesoft/asoftweb/node_modules/.pnpm/esbuild@0.25.9/node_modules/esbuild/lib/main.js:1467
[ng]   let error = new Error(text);
[ng]               ^
[ng] Error: Error during dependency optimization:
[ng] ✘ [ERROR] Could not resolve "@core/event-bus"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:4:20:
[ng]       4 │ import * as i1 from '@core/event-bus';
[ng]         ╵                     ~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/event-bus" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/config/config.service"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:7:49:
[ng]       7 │ ...figService as ConfigService$1 } from '@core/config/config.service';
[ng]         ╵                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/config/config.service" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/directives"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:8:64:
[ng]       8 │ ...andlerDirective, GlobalKeyboardDirective } from '@core/directives';
[ng]         ╵                                                    ~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/directives" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/svg/svg.directive"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:9:29:
[ng]       9 │ import { SvgDirective } from '@core/svg/svg.directive';
[ng]         ╵                              ~~~~~~~~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/svg/svg.directive" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/svg/svg.service"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:10:43:
[ng]       10 │ import { SvgService as SvgService$1 } from '@core/svg/svg.service';
[ng]          ╵                                            ~~~~~~~~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/svg/svg.service" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/types"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:11:69:
[ng]       11 │ ...ype as ElementType$1, ToolType as ToolType$1 } from '@core/types';
[ng]          ╵                                                        ~~~~~~~~~~~~~
[ng]   You can mark the path "@core/types" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/api"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:12:43:
[ng]       12 │ import { ApiService as ApiService$1 } from '@core/api';
[ng]          ╵                                            ~~~~~~~~~~~
[ng]   You can mark the path "@core/api" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/canvas"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:13:49:
[ng]       13 │ import { CanvasService as CanvasService$1 } from '@core/canvas';
[ng]          ╵                                                  ~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/canvas" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/elements"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:14:55:
[ng]       14 │ ... { SelectionService as SelectionService$1 } from '@core/elements';
[ng]          ╵                                                     ~~~~~~~~~~~~~~~~
[ng]   You can mark the path "@core/elements" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/pipes"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:15:69:
[ng]       15 │ ...orPipe, ElementOpacityPipe, PointsToPathPipe } from '@core/pipes';
[ng]          ╵                                                        ~~~~~~~~~~~~~
[ng]   You can mark the path "@core/pipes" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng] ✘ [ERROR] Could not resolve "@core/tools"
[ng]     node_modules/.pnpm/ng-whiteboard@5.0.0_@angular+common@20.3.11_@angular+core@20.3.11_@angular+compiler@20._57316c1536bedbebfa1620d3069d9a07/node_modules/ng-whiteboard/fesm2022/ng-whiteboard.mjs:16:47:
[ng]       16 │ import { ToolsService as ToolsService$1 } from '@core/tools';
[ng]          ╵                                                ~~~~~~~~~~~~~
[ng]   You can mark the path "@core/tools" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
[ng]     at failureErrorWithLog (/mnt/WorkGames/work/abodesoft/asoftweb/node_modules/.pnpm/esbuild@0.25.9/node_modules/esbuild/lib/main.js:1467:15)
[ng]     at /mnt/WorkGames/work/abodesoft/asoftweb/node_modules/.pnpm/esbuild@0.25.9/node_modules/esbuild/lib/main.js:926:25
[ng]     at /mnt/WorkGames/work/abodesoft/asoftweb/node_modules/.pnpm/esbuild@0.25.9/node_modules/esbuild/lib/main.js:1345:9
[ng]     at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
[ng]   errors: [Getter/Setter],
[ng]   warnings: [Getter/Setter]
[ng] }
[ng] Node.js v24.11.0

[ERROR] ng has unexpectedly closed (exit code 1).
```

--- a/README.md
+++ b/README.md
@@ -278,63 +278,63 @@ The whiteboard supports comprehensive keyboard shortcuts for enhanced productivi
 
 ### Selection & Editing
 
-| Shortcut | Action |
-| --- | --- |
-| `Ctrl/Cmd + A` | Select all elements |
-| `Escape` | Clear selection |
-| `Delete` / `Backspace` | Delete selected elements |
-| `Ctrl/Cmd + C` | Copy selected elements |
-| `Ctrl/Cmd + X` | Cut selected elements |
-| `Ctrl/Cmd + V` | Paste elements |
-| `Ctrl/Cmd + D` | Duplicate selected elements |
+| Shortcut               | Action                      |
+| ---------------------- | --------------------------- |
+| `Ctrl/Cmd + A`         | Select all elements         |
+| `Escape`               | Clear selection             |
+| `Delete` / `Backspace` | Delete selected elements    |
+| `Ctrl/Cmd + C`         | Copy selected elements      |
+| `Ctrl/Cmd + X`         | Cut selected elements       |
+| `Ctrl/Cmd + V`         | Paste elements              |
+| `Ctrl/Cmd + D`         | Duplicate selected elements |
 
 ### Undo/Redo
 
-| Shortcut | Action |
-| --- | --- |
-| `Ctrl/Cmd + Z` | Undo last action |
+| Shortcut                                | Action                  |
+| --------------------------------------- | ----------------------- |
+| `Ctrl/Cmd + Z`                          | Undo last action        |
 | `Ctrl/Cmd + Shift + Z` / `Ctrl/Cmd + Y` | Redo last undone action |
 
 ### Layer Management
 
-| Shortcut | Action |
-| --- | --- |
+| Shortcut               | Action         |
+| ---------------------- | -------------- |
 | `Ctrl/Cmd + Shift + ]` | Bring to front |
-| `Ctrl/Cmd + ]` | Bring forward |
-| `Ctrl/Cmd + Shift + [` | Send to back |
-| `Ctrl/Cmd + [` | Send backward |
+| `Ctrl/Cmd + ]`         | Bring forward  |
+| `Ctrl/Cmd + Shift + [` | Send to back   |
+| `Ctrl/Cmd + [`         | Send backward  |
 
 ### Tools
 
-| Shortcut | Action |
-| --- | --- |
-| `V` | Select tool |
-| `P` | Pen tool |
-| `L` | Line tool |
-| `R` | Rectangle tool |
-| `E` | Ellipse tool |
-| `A` | Arrow tool |
-| `T` | Text tool |
-| `I` | Image tool |
-| `H` | Hand tool (pan) |
-| `D` | Eraser tool |
+| Shortcut | Action          |
+| -------- | --------------- |
+| `V`      | Select tool     |
+| `P`      | Pen tool        |
+| `L`      | Line tool       |
+| `R`      | Rectangle tool  |
+| `E`      | Ellipse tool    |
+| `A`      | Arrow tool      |
+| `T`      | Text tool       |
+| `I`      | Image tool      |
+| `H`      | Hand tool (pan) |
+| `D`      | Eraser tool     |
 
 ### Viewport
 
-| Shortcut | Action |
-| --- | --- |
-| `Ctrl/Cmd + +` / `Ctrl/Cmd + =` | Zoom in |
-| `Ctrl/Cmd + -` | Zoom out |
-| `Ctrl/Cmd + 0` | Reset zoom (100%) |
-| `Space + Drag` | Pan canvas (when not in hand tool mode) |
+| Shortcut                        | Action                                  |
+| ------------------------------- | --------------------------------------- |
+| `Ctrl/Cmd + +` / `Ctrl/Cmd + =` | Zoom in                                 |
+| `Ctrl/Cmd + -`                  | Zoom out                                |
+| `Ctrl/Cmd + 0`                  | Reset zoom (100%)                       |
+| `Space + Drag`                  | Pan canvas (when not in hand tool mode) |
 
 ### Modifiers
 
-| Shortcut | Action |
-| --- | --- |
-| `Shift + Drag` | Constrain proportions (drawing shapes) |
-| `Alt/Option + Drag` | Draw from center (shapes) |
-| `Ctrl/Cmd + Drag` | Duplicate while dragging |
+| Shortcut            | Action                                 |
+| ------------------- | -------------------------------------- |
+| `Shift + Drag`      | Constrain proportions (drawing shapes) |
+| `Alt/Option + Drag` | Draw from center (shapes)              |
+| `Ctrl/Cmd + Drag`   | Duplicate while dragging               |
 
 > **Note:** `Ctrl` is used on Windows/Linux, `Cmd` (⌘) is used on macOS.
 
@@ -343,11 +343,13 @@ The whiteboard supports comprehensive keyboard shortcuts for enhanced productivi
 Right-click on the canvas or elements to access the context menu with quick actions:
 
 ### Canvas Context Menu
+
 - **Paste** - Paste copied elements
 - **Select All** - Select all elements on canvas
 - **Clear Canvas** - Remove all elements
 
 ### Element Context Menu
+
 - **Cut** - Cut selected element(s)
 - **Copy** - Copy selected element(s)
 - **Paste** - Paste from clipboard

--- a/eslint.config.cjs
+++ b/eslint.config.cjs
@@ -14,7 +14,7 @@ module.exports = [
         'error',
         {
           enforceBuildableLibDependency: true,
-          allow: ['^.*/eslint(\\.base)?\\.config\\.[cm]?js$', '^@core/'],
+          allow: ['^.*/eslint(\\.base)?\\.config\\.[cm]?js$'],
           depConstraints: [
             {
               sourceTag: '*',

--- a/projects/ng-whiteboard/README.md
+++ b/projects/ng-whiteboard/README.md
@@ -278,63 +278,63 @@ The whiteboard supports comprehensive keyboard shortcuts for enhanced productivi
 
 ### Selection & Editing
 
-| Shortcut | Action |
-| --- | --- |
-| `Ctrl/Cmd + A` | Select all elements |
-| `Escape` | Clear selection |
-| `Delete` / `Backspace` | Delete selected elements |
-| `Ctrl/Cmd + C` | Copy selected elements |
-| `Ctrl/Cmd + X` | Cut selected elements |
-| `Ctrl/Cmd + V` | Paste elements |
-| `Ctrl/Cmd + D` | Duplicate selected elements |
+| Shortcut               | Action                      |
+| ---------------------- | --------------------------- |
+| `Ctrl/Cmd + A`         | Select all elements         |
+| `Escape`               | Clear selection             |
+| `Delete` / `Backspace` | Delete selected elements    |
+| `Ctrl/Cmd + C`         | Copy selected elements      |
+| `Ctrl/Cmd + X`         | Cut selected elements       |
+| `Ctrl/Cmd + V`         | Paste elements              |
+| `Ctrl/Cmd + D`         | Duplicate selected elements |
 
 ### Undo/Redo
 
-| Shortcut | Action |
-| --- | --- |
-| `Ctrl/Cmd + Z` | Undo last action |
+| Shortcut                                | Action                  |
+| --------------------------------------- | ----------------------- |
+| `Ctrl/Cmd + Z`                          | Undo last action        |
 | `Ctrl/Cmd + Shift + Z` / `Ctrl/Cmd + Y` | Redo last undone action |
 
 ### Layer Management
 
-| Shortcut | Action |
-| --- | --- |
+| Shortcut               | Action         |
+| ---------------------- | -------------- |
 | `Ctrl/Cmd + Shift + ]` | Bring to front |
-| `Ctrl/Cmd + ]` | Bring forward |
-| `Ctrl/Cmd + Shift + [` | Send to back |
-| `Ctrl/Cmd + [` | Send backward |
+| `Ctrl/Cmd + ]`         | Bring forward  |
+| `Ctrl/Cmd + Shift + [` | Send to back   |
+| `Ctrl/Cmd + [`         | Send backward  |
 
 ### Tools
 
-| Shortcut | Action |
-| --- | --- |
-| `V` | Select tool |
-| `P` | Pen tool |
-| `L` | Line tool |
-| `R` | Rectangle tool |
-| `E` | Ellipse tool |
-| `A` | Arrow tool |
-| `T` | Text tool |
-| `I` | Image tool |
-| `H` | Hand tool (pan) |
-| `D` | Eraser tool |
+| Shortcut | Action          |
+| -------- | --------------- |
+| `V`      | Select tool     |
+| `P`      | Pen tool        |
+| `L`      | Line tool       |
+| `R`      | Rectangle tool  |
+| `E`      | Ellipse tool    |
+| `A`      | Arrow tool      |
+| `T`      | Text tool       |
+| `I`      | Image tool      |
+| `H`      | Hand tool (pan) |
+| `D`      | Eraser tool     |
 
 ### Viewport
 
-| Shortcut | Action |
-| --- | --- |
-| `Ctrl/Cmd + +` / `Ctrl/Cmd + =` | Zoom in |
-| `Ctrl/Cmd + -` | Zoom out |
-| `Ctrl/Cmd + 0` | Reset zoom (100%) |
-| `Space + Drag` | Pan canvas (when not in hand tool mode) |
+| Shortcut                        | Action                                  |
+| ------------------------------- | --------------------------------------- |
+| `Ctrl/Cmd + +` / `Ctrl/Cmd + =` | Zoom in                                 |
+| `Ctrl/Cmd + -`                  | Zoom out                                |
+| `Ctrl/Cmd + 0`                  | Reset zoom (100%)                       |
+| `Space + Drag`                  | Pan canvas (when not in hand tool mode) |
 
 ### Modifiers
 
-| Shortcut | Action |
-| --- | --- |
-| `Shift + Drag` | Constrain proportions (drawing shapes) |
-| `Alt/Option + Drag` | Draw from center (shapes) |
-| `Ctrl/Cmd + Drag` | Duplicate while dragging |
+| Shortcut            | Action                                 |
+| ------------------- | -------------------------------------- |
+| `Shift + Drag`      | Constrain proportions (drawing shapes) |
+| `Alt/Option + Drag` | Draw from center (shapes)              |
+| `Ctrl/Cmd + Drag`   | Duplicate while dragging               |
 
 > **Note:** `Ctrl` is used on Windows/Linux, `Cmd` (⌘) is used on macOS.
 
@@ -343,11 +343,13 @@ The whiteboard supports comprehensive keyboard shortcuts for enhanced productivi
 Right-click on the canvas or elements to access the context menu with quick actions:
 
 ### Canvas Context Menu
+
 - **Paste** - Paste copied elements
 - **Select All** - Select all elements on canvas
 - **Clear Canvas** - Remove all elements
 
 ### Element Context Menu
+
 - **Cut** - Cut selected element(s)
 - **Copy** - Copy selected element(s)
 - **Paste** - Paste from clipboard

--- a/projects/ng-whiteboard/src/lib/core/components/canvas/whiteboard-canvas.component.ts
+++ b/projects/ng-whiteboard/src/lib/core/components/canvas/whiteboard-canvas.component.ts
@@ -8,17 +8,17 @@ import {
   inject,
   ViewChild,
 } from '@angular/core';
-import { ConfigService } from '@core/config/config.service';
-import { GlobalKeyboardDirective, ResizeHandlerDirective } from '@core/directives';
+import { ConfigService } from '../../config/config.service';
+import { GlobalKeyboardDirective, ResizeHandlerDirective } from '../../directives';
 
-import { SvgDirective } from '@core/svg/svg.directive';
-import { SvgService } from '@core/svg/svg.service';
-import { ElementType, ToolType, WhiteboardElement } from '@core/types';
-import { ApiService } from '@core/api';
-import { CanvasService } from '@core/canvas';
-import { SelectionService } from '@core/elements';
-import { GripCursorPipe, ElementOpacityPipe, PointsToPathPipe } from '@core/pipes';
-import { ToolsService } from '@core/tools';
+import { SvgDirective } from '../../svg/svg.directive';
+import { SvgService } from '../../svg/svg.service';
+import { ElementType, ToolType, WhiteboardElement } from '../../types';
+import { ApiService } from '../../api';
+import { CanvasService } from '../../canvas';
+import { SelectionService } from '../../elements';
+import { GripCursorPipe, ElementOpacityPipe, PointsToPathPipe } from '../../pipes';
+import { ToolsService } from '../../tools';
 
 @Component({
   selector: 'ng-whiteboard-canvas',

--- a/projects/ng-whiteboard/src/lib/core/elements/selection.service.ts
+++ b/projects/ng-whiteboard/src/lib/core/elements/selection.service.ts
@@ -1,5 +1,5 @@
 import { computed, Injectable, signal, Signal } from '@angular/core';
-import { EventBusService } from '@core/event-bus';
+import { EventBusService } from '../event-bus';
 import { CanvasService } from '../canvas/canvas.service';
 import { ClipboardService } from '../input/clipboard.service';
 import { ToolsService } from '../tools/tools.service';

--- a/projects/ng-whiteboard/src/lib/core/pipes/points-to-path.pipe.ts
+++ b/projects/ng-whiteboard/src/lib/core/pipes/points-to-path.pipe.ts
@@ -1,7 +1,7 @@
 import { Pipe, PipeTransform } from '@angular/core';
 import { StrokeOptions } from '../utils/drawing';
-import { getStrokePoints } from '@core/utils/drawing/stroke-points';
-import { getSvgPathFromStroke } from '@core/utils/drawing/path';
+import { getStrokePoints } from '../utils/drawing/stroke-points';
+import { getSvgPathFromStroke } from '../utils/drawing/path';
 
 @Pipe({
   name: 'pointsToPath',

--- a/projects/ng-whiteboard/src/lib/core/tools/tools.service.spec.ts
+++ b/projects/ng-whiteboard/src/lib/core/tools/tools.service.spec.ts
@@ -1,5 +1,4 @@
 import { TestBed } from '@angular/core/testing';
-import { signal } from '@angular/core';
 import { ToolsService } from './tools.service';
 import { ToolFactory } from './tool-factory.service';
 import { EventBusService } from '../event-bus/event-bus.service';
@@ -118,7 +117,8 @@ describe('ToolsService', () => {
       // Create a new service instance via TestBed but don't set ApiService
       const newService = TestBed.inject(ToolsService);
       // Clear the apiServiceCache to simulate not being set
-      (newService as any).apiServiceCache = undefined;
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (newService as any)._apiServiceCache.set(undefined);
 
       expect(() => newService.getToolInstance(ToolType.Hand)).toThrow(
         'ApiService not set. Call setApiService() first.'
@@ -136,6 +136,8 @@ describe('ToolsService', () => {
     it('should throw error when no active tool instance', () => {
       // Create a service and manually clear the current tool instance
       const testService = TestBed.inject(ToolsService);
+      // Reset the signal to null for testing
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       (testService as any)._currentToolInstance.set(null);
 
       expect(() => testService.getActiveToolInstance()).toThrow('No active tool instance found.');
@@ -323,12 +325,16 @@ describe('ToolsService', () => {
     it('should enable/disable tool', () => {
       const configs = service.getToolConfigs();
       const handConfig = configs.find((c) => c.type === ToolType.Hand);
-      expect(handConfig).toBeDefined();
 
-      const result = service.setToolEnabled(handConfig!.id, false);
+      if (!handConfig) {
+        fail('Hand config should be defined');
+        return;
+      }
+
+      const result = service.setToolEnabled(handConfig.id, false);
       expect(result).toBe(true);
 
-      const updatedConfig = service.getToolConfig(handConfig!.id);
+      const updatedConfig = service.getToolConfig(handConfig.id);
       expect(updatedConfig?.enabled).toBe(false);
     });
 
@@ -341,8 +347,13 @@ describe('ToolsService', () => {
       const configs = service.getToolConfigs();
       const handConfig = configs.find((c) => c.type === ToolType.Hand);
 
+      if (!handConfig) {
+        fail('Hand config should be defined');
+        return;
+      }
+
       service.setActiveTool(ToolType.Hand);
-      service.setToolEnabled(handConfig!.id, false);
+      service.setToolEnabled(handConfig.id, false);
 
       expect(service.getActiveToolType()).toBe(ToolType.Pen);
     });
@@ -366,10 +377,15 @@ describe('ToolsService', () => {
       const configs = service.getToolConfigs();
       const handConfig = configs.find((c) => c.type === ToolType.Hand);
 
-      service.setToolEnabled(handConfig!.id, false);
+      if (!handConfig) {
+        fail('Hand config should be defined');
+        return;
+      }
+
+      service.setToolEnabled(handConfig.id, false);
       const available = service.availableTools();
 
-      expect(available.some((t) => t.id === handConfig!.id)).toBe(false);
+      expect(available.some((t) => t.id === handConfig.id)).toBe(false);
     });
   });
 

--- a/projects/ng-whiteboard/src/lib/core/tools/tools.service.ts
+++ b/projects/ng-whiteboard/src/lib/core/tools/tools.service.ts
@@ -110,7 +110,7 @@ const DEFAULT_TOOLS: ReadonlyArray<ToolConfig> = [
 export class ToolsService {
   private readonly eventBusService = inject(EventBusService);
   private readonly toolFactory = inject(ToolFactory);
-  private apiServiceCache?: ApiService;
+  private readonly _apiServiceCache = signal<ApiService | undefined>(undefined);
 
   private readonly _selectedTool = signal<ToolType>(ToolType.Pen);
   private readonly _toolConfigs = signal<ReadonlyMap<string, ToolConfig>>(new Map());
@@ -143,14 +143,21 @@ export class ToolsService {
     effect(
       () => {
         const toolType = this.effectiveTool();
-        this.updateCurrentToolInstance(toolType);
+        const apiService = this._apiServiceCache();
+        // Only update tool instance after ApiService is initialized
+        if (apiService) {
+          this.updateCurrentToolInstance(toolType);
+        }
       },
       { allowSignalWrites: true }
     );
 
     effect(
       () => {
-        this.updateCursorForActiveTool();
+        // Only update cursor after tool instance is available
+        if (this._currentToolInstance()) {
+          this.updateCursorForActiveTool();
+        }
       },
       { allowSignalWrites: true }
     );
@@ -219,10 +226,11 @@ export class ToolsService {
     }
 
     if (!this.toolInstanceCache.has(toolType)) {
-      if (!this.apiServiceCache) {
+      const apiService = this._apiServiceCache();
+      if (!apiService) {
         throw new Error('ApiService not set. Call setApiService() first.');
       }
-      const tool = this.toolFactory.createTool(toolType, this.apiServiceCache);
+      const tool = this.toolFactory.createTool(toolType, apiService);
       this.toolInstanceCache.set(toolType, tool);
     }
 
@@ -393,6 +401,6 @@ export class ToolsService {
   }
 
   setApiService(apiService: ApiService): void {
-    this.apiServiceCache = apiService;
+    this._apiServiceCache.set(apiService);
   }
 }

--- a/projects/ng-whiteboard/src/lib/ng-whiteboard.component.ts
+++ b/projects/ng-whiteboard/src/lib/ng-whiteboard.component.ts
@@ -43,6 +43,19 @@ import { PanService, WheelHandlerService, ZoomService } from './core/viewport';
       <wb-context-menu></wb-context-menu>
     </div>
   `,
+  styles: [
+    `
+      :host {
+        display: block;
+        width: inherit;
+        height: inherit;
+        min-width: inherit;
+        min-height: inherit;
+        max-width: inherit;
+        max-height: inherit;
+      }
+    `,
+  ],
   providers: [
     SvgService,
     ApiService,

--- a/tsconfig.base.json
+++ b/tsconfig.base.json
@@ -15,7 +15,6 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@core/*": ["projects/ng-whiteboard/src/lib/core/*"],
       "ng-whiteboard": ["projects/ng-whiteboard/src/index.ts"]
     }
   },

